web/src/utils/constants.ts

```diff
@@ -77,3 +77,106 @@ export const timeAxisMapping: Record<TimeAverages, keyof Duration> = {
   monthly: 'months',
   yearly: 'years',
 };
+
+export const sourceLinkMapping: { [key: string]: string } = {
```

Comment:
Do we want to keep this as a JSON file instead?
We keep other config mappings as JSON makes sense to do the same here?

Reply:
@madsnedergaard What do you say?

Reply:
This discussion should probably take place in #6786

Reply:
[Moved here](https://github.com/electricitymaps/electricitymaps-contrib/pull/6786#discussion_r1625698767)